jaseci_core/jaseci/actions/standard/file.py

```diff
@@ -40,3 +42,38 @@ def dump_json(fn: str, obj, indent: int = None):
     """Standard built in for dumping json to file from dictionary"""
     with open(fn, 'w') as file:
         json.dump(obj, file, indent=indent)
+
+
+@jaseci_action()
+def build_form_data_from_b64(files: list):
+    """Standard built in for building form data from base64 file list"""
+    """BytesIO avoids file writing on server"""
+    ret = []
+    for f in files:
+        ret.append(
+            ('file', (
+                f["name"],
+                BytesIO(
+                    b64decode(f["base64"])
+                )
+            ))
+        )
+
+    return ret
+
+@jaseci_action()
+def load_url(url: str, header: dict):
+    """Standard built in for download file from url"""
+
+    with requests.get(url, stream = True, headers = header) as res:
+        res.raise_for_status()
+        with BytesIO() as buffer:
+            for chunk in res.iter_content(chunk_size=8192):
+                buffer.write(chunk)
+            ret = buffer.getvalue()
+            
+    return ret
+
+@jaseci_action()
+def bytes_to_base64(file: bytes, encoding: str = "utf-8"):
```

Comment:
Similar question here based on the scope of use case with this lib call. Lets hop on a quick call :-)

Reply:
This is for converting the file's content from the 3rd party service. When someone tries to download their uploaded file,
the 3rd party service's API will return it as bytes reason for converting it back to base64 so I can include it on the json response.